script/build.py

```diff
@@ -44,6 +44,7 @@ def main():
     ]
```

Comment:
Please, change the version to `m105-adda216f-3` here:
https://github.com/JetBrains/skia-pack/blob/main/.github/workflows/build.yml#L16

Reply:
ok, I have changed, please review